demos/kdf/argon2.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
```

Comment:
Maybe add something like this..

````
/*
 * Example showing how to use Argon2 KDF.
 * See doc/man7/EVP_KDF-ARGON2.pod for more information.
 */
````

Reply:
I just added "man EVP_KDF-ARGON2" as I expect man pages are installed.